canonical-web-and-design/ubuntu.com
# fix forms in IE11

## Done

- Included a polyfill.io script tag that handles all the JS IE11 otherwise can't 
  - Open to suggestions on this, I'm not convinced it's a long term fix, mostly because:
    - it's another network request
    - I'm pretty sure we could setup webpack/babel to do it for us
    - we could just (within reason) not write JS that requires polyfilling/heavy babel configuration
  - but it does what we need in the short term. Plus, the polyfill.io API is smart enough to only return polyfills for browsers that need them, so for most users this script will only be a few bytes in size, and basically empty.
- Updated the version of `cookie-policy` to one that doesn't include ES6 syntax

## QA

- View [the demo](https://ubuntu-com-canonical-web-and-design-pr-6689.run.demo.haus/openstack/managed) in Browserstack, IE11
- Try not to cry at what IE11 does to our nicely designed layouts
- Click "Talk to us" button in the hero strip
- See that the modal launches and you can successfully complete the form
- Try other forms on the site, see that they can be submitted successfully


## Issue / Card

Fixes #6656
